mrp_workcenter_segmentation/tests/test_workcenter_segmentation.py

```diff
@@ -73,6 +73,14 @@ def create_wizard(self, mrp_production, qty=1):
         wz_brw.do_produce()
         return True
 
+    def get_account_values(self, account_name):
+        rec_ids = self.mrp_production_e.account_move_id.line_id.filtered(
+            lambda l: l.account_id.name == account_name)
+        res = []
+        for rec_id in rec_ids:
+            res.append((rec_id.debit, rec_id.credit))
```

Comment:
I would be more functionally valuable to gather the values of the debit and credit into variables
than passing the whole list of values.

that is 

res = (account='ACCOUNT', debit=200, credit=100)
return res

and so on.